yt_dlp/extractor/globo.py

```diff
@@ -38,126 +33,70 @@ class GloboIE(InfoExtractor):
-            'uploader': 'Rede Globo',
-            'uploader_id': '196',
+            'uploader': 'Bom Dia Brasil',
+            'uploader_id': '810',
         },
         'params': {
             'skip_download': True,
         },
-    }, {
-        'url': 'http://canalbrasil.globo.com/programas/sangue-latino/videos/3928201.html',
-        'only_matching': True,
-    }, {
-        'url': 'http://globosatplay.globo.com/globonews/v/4472924/',
-        'only_matching': True,
-    }, {
-        'url': 'http://globotv.globo.com/t/programa/v/clipe-sexo-e-as-negas-adeus/3836166/',
-        'only_matching': True,
-    }, {
-        'url': 'http://globotv.globo.com/canal-brasil/sangue-latino/t/todos-os-videos/v/ator-e-diretor-argentino-ricado-darin-fala-sobre-utopias-e-suas-perdas/3928201/',
-        'only_matching': True,
-    }, {
-        'url': 'http://canaloff.globo.com/programas/desejar-profundo/videos/4518560.html',
-        'only_matching': True,
     }, {
         'url': 'globo:3607726',
         'only_matching': True,
-    }, {
-        'url': 'https://globoplay.globo.com/v/10248083/',
-        'info_dict': {
-            'id': '10248083',
-            'ext': 'mp4',
-            'title': 'Melhores momentos: Equador 1 x 1 Brasil pelas Eliminatórias da Copa do Mundo 2022',
-            'duration': 530.964,
-            'uploader': 'SporTV',
-            'uploader_id': '698',
-        },
-        'params': {
-            'skip_download': True,
-        },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        self._request_webpage(
-            HEADRequest('https://globo-ab.globo.com/v2/selected-alternatives?experiments=player-isolated-experiment-02&skipImpressions=true'),
-            video_id, 'Getting cookies')
-
+        video_view = '''
+        query getVideoView($videoId: ID!) {
+            video(id: $videoId) {
+                duration
+                description
+                headline
+                title {
+                    originProgramId
+                    headline
+                }
+            }
+        }
+        '''
         video = self._download_json(
-            f'http://api.globovideos.com/videos/{video_id}/playlist',
-            video_id)['videos'][0]
-        if not self.get_param('allow_unplayable_formats') and video.get('encrypted') is True:
-            self.report_drm(video_id)
-
-        title = video['title']
+            f'https://cloud-jarvis.globo.com/graphql?operationName=getVideoView&variables=%7B"videoId":"{video_id}"%7D&query={video_view}', video_id,
+            headers={'content-type': 'application/json', 'x-platform-id': 'web', 'x-device-id': 'desktop', 'x-client-version': '2024.12-5'})['data']['video']
+        title = video['headline']
+        uploader = video['title'].get('headline')
+        uploader_id = str_or_none(video['title'].get('originProgramId'))
 
         formats = []
         security = self._download_json(
```

Comment:
I don't like continuing to name this variable `security` when we've removed all the security hash code. I changed it in my patch for a reason